sotodlib/io/imprinter.py

```diff
@@ -269,7 +288,7 @@ def bind_book(self, book, session=None, output_root="out"):
         try:
             readout_ids = self.get_readout_ids_for_book(book)
         except ValueError:
-            pass
+            readout_ids = None
```

Comment:
will this still make books just with no readout ids? Asking because I don't think we want to make books without readout ids. Instead we should skip over them and then trigger an alert of some type.

Reply:
yes that's what's it currently doing, but I've now remove the error handling here